stix_shifter_modules/secretserver/stix_translation/json/stix_2_1/event_from_stix_map.json

```diff
@@ -0,0 +1,35 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": ["IpAddress", "Server"],
+       "src_ip": ["IpAddress"],
+        "dst_ip": ["Server"]
+    }
+  },
+  "url": {
+    "fields": {
+      "value": [
+        "url"
+      ]
+    }
+  },
+
+  "x-secret": {
+    "fields": {
+      "secret_name": ["SecretName"],
+      "username": ["EmailAddress"],
+      "secret_id": ["ItemId"],
+      "user_id": ["UserId"],
+      "server_user_name" : ["Username"]
```

Comment:
This should be in a `user-account:display_name` property

Reply:
We have done our db parser changes with the above mapping. If it is not mandatory this time we can proceed.